dll/win32/shell32/shellrecyclebin/recyclebin_v5.c

```diff
@@ -512,6 +512,11 @@ RecycleBin5_RecycleBin5_Restore(
                 return E_FAIL;
             }
 
+            // Delete the file on the recycle bin as it's actually restored
+            op.wFunc = FO_DELETE;
+            op.fFlags = FOF_SILENT | FOF_NOCONFIRMATION | FOF_NOERRORUI | FOF_NOCONFIRMMKDIR;
+            SHFileOperationW(&op);
```

Comment:
I wonder, actually, whether we could use `FO_MOVE` in the "restore file" operation above.
@hpoussin do you have any insight on that? (the pros and cons)

Reply:
well, it would be more useful a move

Reply:
it easy to change that, i mean it's not logical to store the file that we copied and restored, that would be an incremental space consumption

Reply:
Totally done